db/migrate/20160114121308_remove_link_fields.rb

```diff
@@ -0,0 +1,18 @@
+class RemoveLinkFields < ActiveRecord::Migration
```

Comment:
The script is here: https://github.com/alphagov/policy-publisher/pull/124

Can I remove this migration from this pull request?